docs/ Added sub topic about running multiple bots

JeremyKono opened the pull request:
Removed how to run multiple bots from advanced configuration and made a sub topic under installation for better visibility
<img width="952" alt="hummingbot docs" src="https://user-images.githubusercontent.com/73882610/100860932-1b3eb800-34cc-11eb-9638-8be260f52b31.png">

Also added how to run multiple bots via Binary for users who run Hummingbot on binary
<img width="948" alt="Running multiple bots" src="https://user-images.githubusercontent.com/73882610/100860675-c26f1f80-34cb-11eb-8e68-6e2028c850a3.png">

dennisocana reviewed (changes requested):
In this gif the other HB window did not open rather, it searched in Microsoft Edge.

![image](https://user-images.githubusercontent.com/50150287/100997977-ebf47d80-3595-11eb-976c-292f121dd5d5.png)

JeremyKono commented:
> In this gif the other HB window did not open rather, it searched in Microsoft Edge.
> 
> ![image](https://user-images.githubusercontent.com/50150287/100997977-ebf47d80-3595-11eb-976c-292f121dd5d5.png)

Changed the .gif I added into the docs and showed side-by-side both hummingbot terminals 
![multiple_bots1](https://user-images.githubusercontent.com/73882610/101043960-d1340000-35b9-11eb-9413-10756a5e3298.gif)
